docs/topics/modules/using_stack_report.adoc

```diff
@@ -1,62 +1,62 @@
 [id="using_stack_report"]
 = Using your Stack Report
 
-{osio} Analytics helps you make informed choices with respect to your development stack and its component libraries by analyzing your stack at the development stage, within your IDE, and at the deployment stage.
+{osio} Analytics helps you make informed choices with respect to your development stack and its dependencies by analyzing your stack at the development stage, within your IDE, and at the deployment stage.
 
 == Development Stage
-{osio} Analytics analyzes your stack at the development stage and flags components with security vulnerabilities in your Che workspace. It also recommends a secure, alternate version for your stack.
+{osio} provides an integrated development environment in the form of Eclipse Che workspace to develop your code.
+{osio} Analytics flags dependencies with security vulnerabilities by analyzing your stack, while you modify it, at the development stage within your Che workspace. It also suggests a secure, alternate version of the dependency for your stack.
 
-.Prerequisite
+.Prerequsite
 
-Ensure that you use link:e2e_workshop.html#creating_che_workspace-hello-world[Che workspace] to develop an application.
+. Create a link:e2e_workshop.html#creating_che_workspace-hello-world[Che workspace] to develop your stack.
```

Comment:
The file `e2e_workshop.html` no longer exists. It's now `getting-started-guide.html`.